Chief Election Officials

putorti opened the pull request:
- Update Lieutenant Governors, Secretaries of State, and Chief Election Officers as needed

johnseekins commented:
Hi @putorti . The last error is actually:
```
Nancy-Dahlstrom-d6b940121-937e-47ba-916a-626b67fe54f4.yml
   id: must match ocd-person/UUID format
```
Which is a file you added.

I'm not actually sure what this error _means_ (as I don't see anything wrong with the file), but the error is in the files you've added/changed, not elsewhere.

putorti commented:
> must match ocd-person/UUID format

I fixed the mismatch [here](https://github.com/openstates/people/pull/1138/commits/a182da5f06a54ecaa4b6b9b0a7aa00d6a5fa0b5d), but it was the wrong OCD, fixed that [here](https://github.com/openstates/people/pull/1138/commits/dcd6fbf6aa823d229720e1d3124603701abd5420). Let me know if there's anything else, thank you!
